cime/src/drivers/mct/cime_config/namelist_definition_drv.xml

```diff
@@ -4749,4 +4749,18 @@
     </values>
   </entry>
 
+  <entry id="seq_flux_atmocn_minwind">
+    <type>real</type>
+    <category>seq_flux_mct</category>
+    <group>seq_flux_mct_inparm</group>
+    <desc>
+       minimum wind speed for atmOcn flux calculations
+    </desc>
+    <values>
+      <value>0.5</value>
+      <value cime_model='cesm' aqua_planet_sst_type="sst_aquap11">1.0</value>
+      <value cime_model='e3sm' aqua_planet_sst_type="sst_aquap_constant">1.0</value>
+    </values>
```

Comment:
This looks good for E3SM, but CESM is also going to use "sst_aquap_constant", so it should be the same for both models.

Reply:
@whannah1 , done